survey/dashboards.py

```diff
@@ -0,0 +1,1708 @@
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '0px',
+                                    'fontWeight': 'normal',
+                                    'textAlign': 'center',
+                                })
+                    ])
+                ], style={
+                    'borderRadius': '10px',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)'
+                })
+            ], width=4),
+
+            # Completion Rate
+            dbc.Col([
+                dbc.Card([
+                    dbc.CardBody([
+                        html.H6("Completion Rate",
+                                style={
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '0px',
+                                    'textAlign': 'center'
+                                }),
+                        html.H3(id='completion-rate',
+                                style={
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '0px',
+                                    'fontWeight': 'normal',
+                                    'textAlign': 'center'
+                                })
+                    ])
+                ], style={
+                    'borderRadius': '10px',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)'
+                })
+            ], width=4),
+
+            # Average Score
+            dbc.Col([
+                dbc.Card([
+                    dbc.CardBody([
+                        html.H6("Average Score",
+                                style={
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '8px',
+                                    'textAlign': 'center'
+                                }),
+                        html.H3(id='average-score',
+                                style={
+                                    'fontFamily': THEME['font_family'],
+                                    'color': THEME['text_color'],
+                                    'marginBottom': '0px',
+                                    'fontWeight': 'normal',
+                                    'textAlign': 'center'
+                                })
+                    ])
+                ], style={
+                    'borderRadius': '10px',
+                    'boxShadow': '0 2px 4px rgba(0, 0, 0, 0.1)'
+                })
+            ], width=4)
+        ], className='g-4', style={'marginBottom': '20px'}),
+
+        # Section A
+        dbc.Row([
```

Comment:
Maybe use a loop for make a separate function to generate each section? There's repetition of code for all the sections.